#! /usr/bin/env python

PACKAGE='tod_video'

from dynamic_reconfigure.parameter_generator_catkin import *

class SensorLevels:
    RECONFIGURE_RUNNING = 0
    RECONFIGURE_STOP = 1

gen = ParameterGenerator()
# Name    Type    Reconfiguration Level    Description    Default    Min    Max
gen.add("camera_name", str_t, SensorLevels.RECONFIGURE_RUNNING, "Name of Camera.", "")
gen.add("pause", bool_t, SensorLevels.RECONFIGURE_RUNNING, "Turn on and off stream for camera.", False)

exit(gen.generate(PACKAGE, "tod_video", "Client"))
